docs/getting-started/your-first-set-of-documentation.rst

```diff
@@ -96,22 +96,22 @@ DocBlocks are divided into the following three parts. Each of these parts is opt
 may not exist without a Summary.
 
 Summary
-  Sometimes called a short description, provides a brief introduction into the function of the associated element.
-  A Summary ends
-  in one of these situations:
+  The Summary, sometimes called a short description, provides a brief introduction into the function
+  of the associated element. A Summary ends when it encounters either of the below situations::
 
-    1. A dot is following by a line break, or
-    2. Two subsequent line breaks are encountered.
+    1. A period ``.``, followed by a line break, or
```

Comment:
I think the `1. / 2.` implies a list that itself implies _do **all** of these_ rather than _either / or_, so maybe switch to `*`

```
* a period `.` followed by a line break
* or a blank comment line
```